nbs/examples/Multivariate_with_TSMixer.ipynb

```diff
@@ -16,11 +16,11 @@
    "source": [
```

Comment:
<p>same here, I only see the MAE and MSE for MLPMultivariate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/938/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Should be good now</p>